assets/settings/default.json

```diff
@@ -556,7 +556,7 @@
     // Position of the close button on the editor tabs.
     "close_position": "right",
     // Whether to show the file icon for a tab.
-    "file_icons": false,
+    "file_icons": true,
```

Comment:
Not showing file icons in tabs by default was an intentional decision, and I don't think we want to change it.

Reply:
I can revert, but curious as to why? file icons are already set to true in the project panel, the outline panel and the file finder.